workspaces/core/src/banner/private/Multiline.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+export interface Props {
+  description: string;
+  title?: string;
+  iconHeader?: React.ReactElement;
+  buttons?: React.ReactElement[];
```

Comment:
I would prefer to have one prop per button. I don’t think that we will have more than 2 buttons in the future
`discard: React.ReactElement;
  apply: React.ReactElement;`

Reply:
@yarema184  your suggestion is not backwards compatible, as current version of banner has this single property. but if you insist on changing this i'm totally fine with that as well

![32](https://user-images.githubusercontent.com/58221984/177655413-34b5f289-59a4-4939-805a-1a5fa06c891a.jpg)